### DISCUSSION TITLE
Failed to check realizability

### DISCUSSION THREAD

**Discussion by lht011129** [NASA-SW-VnV/fret#81] [Q&A]
I have downloaded jkind.jar, jkind, jrealizability and jlustre2kind from the the latest release and changed permissions for the downloaded files to be executable, as figure1 showed. I have added the directory to PATH environment variable. But when I want to check realizability in FRET, I met the issue showed in figure2. 
![image](https://github.com/NASA-SW-VnV/fret/assets/84820163/b3c4ce54-0164-45e0-b0f3-9edba74e5231)
![image](https://github.com/NASA-SW-VnV/fret/assets/84820163/11909792-a122-4666-bda0-040287322c1b)

**Comment by andreaskatis** [comment]
Hi @lht011129,

Jkind (or Kind 2) is one of two required dependencies for realizability checking. The other one is Z3, which also needs to be added to PATH. Do you have Z3 installed?

Best Regards,

Andreas

**Comment by lht011129** [comment]
Hi Andreas,


Thanks for your suggestion, I have resolved this issue by installing Z3.


Best Regards,


Lin




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NASA-SW-VnV/fret"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月15日(星期三) 晚上7:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NASA-SW-VnV/fret] Failed to check realizability (Discussion #81)





 
Hi @lht011129,
 
Jkind (or Kind 2) is one of two required dependencies for realizability checking. The other one is Z3, which also needs to be added to PATH. Do you have Z3 installed?
 
Best Regards,
 
Andreas

**Comment by andreaskatis** [comment]
Thank you for letting us know!
